src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -200,15 +208,13 @@ private Card createJobCompletedCard() {
             } else if (result == Result.UNSTABLE) {
                 status = "Build Unstable";
                 summary += " Unstable";
-            } else if (result == Result.SUCCESS) {
-                status = "Build Success";
-                summary += " Success";
             } else if (result == Result.NOT_BUILT) {
                 status = "Not Built";
                 summary += " Not Built";
             } else {
                 // if we are here it means that something went wrong in logic above
                 // and we are facing unsupported status or case
+                log("Unsupported result: " + result);
```

Comment:
if the logic above is correct and there is no new Jenkins result - this code should never happen. If only...